bees/ircbee/ircbee.go

```diff
@@ -156,6 +156,55 @@ func (mod *IrcBee) Run(eventChan chan bees.Event) {
 	mod.client.HandleFunc("disconnected", func(conn *irc.Conn, line *irc.Line) {
 		mod.connectedState <- false
 	})
+
+	statusChange := func (conn *irc.Conn, line *irc.Line) {
```

Comment:
I would declare this method outside of Run(). Makes the code easier to read and the inlining provides no advantages here.